Bottom: b5c6f0de91afaf7f72b38a0a06ce98ddf6b22bd2
Top:    4dd0962ab5e670584c37d9a2aea4fdb9aae71a9f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 17:41:22 +0000

Refresh of debug-wonky-copper-polygon

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 25da812..a324066 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1221,7 +1221,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL));
 
-
     break;
   }
